sbi/inference/abc/mcabc.py

```diff
@@ -90,8 +92,10 @@ def __call__(
                 sass regression, default 1 - no expansion.
 
         Returns:
-            posterior: Empirical distribution based on selected parameters.
-            distances: Tensor of distances of the selected parameters.
+            theta: accepted parameters
```

Comment:
Given that this has apparently been causing confusion, it might be good to extend the documentation here a bit.

Reply:
Also, it seems that what is returned depends on the value of `kde`. Would be good to document this.